eng/packages/plugins/client/Client.Plugin/test/Client.Plugin.Tests.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <Version>1.0.0-beta.1</Version>
+    <TargetFrameworks>net9.0</TargetFrameworks>
```

Comment:
Should this reference the required attribute so that we're not specifying twice?    No real harm since there's great locality to them, just curious.